handlers/BES_Modules_The_HDF5_Handler.adoc

```diff
@@ -143,30 +478,393 @@ H5.IgnoreUnknownTypes is true. See above.
+    This key and the following associated keys will help mitigate these issues. 
+  ** If this key is set to __true__, only compressed HDF5 variables are cached. If compressed variables
+  are cached, there is no data decompression time when retrieving the data. Therefore, performance may get improved. 
+  ** The following keys are provided to further limit the compressed variables of which the data is cached to the disk when the H5.DiskCacheComp is set to __true__.
+  *** H5.DiskCacheFloatOnlyComp: If this key is set to __true__, only floating-point compressed variables are cached.
+  *** H5.DiskCacheCompThreshold: To take advantage of this key its value must be a floating-point number greater than 1. 
+  **** The handler will compare the compression ratio of a variable with this number, 
+  only when the compression ratio is smaller than this number(that is: the variable is hard to compress), the variable is cached.
+  In other words, hard compressed variable usually takes longer decompression time.
+  So using disk cache may greatly reduce the processing time.
+  *** H5.DiskCacheCompVarSize: The value of this key represents the variable size in kilobytes. It must be a positive integer number. 
+  **** Only if the (uncompressed) variable size that is greater than this value, that variable data is cached.
+  For example, if this number is 100, only the size of variable that is >100K will be cached.
+  
+  
+=== Keys for Default Option
+H5.DefaultHandleDimension::
+When this key is set to __true__, the handler follows the netCDF-4 data model to handle the HDF5 dimensions if possible.
+
+=== Default BES Key Values
+This is the default setting for BES keys in Hyrax 1.16.5. It means that even without setting any BES key values, the handler will generate either DAP2 or DAP4 output as if these BES key values are set. As the software improves, the default setting may change; check the HDF5 handler configuration file https://github.com/OPENDAP/bes/blob/master/modules/hdf5_handler/h5.conf.in[h5.conf.in] at github. 
+
+----
+H5.EnableCF=true
+H5.EnableCFDMR=true
+H5.ForceFlattenNDCoorAttr=true
+H5.EnableCoorattrAddPath=true
+H5.EnableDAP4Coverage=true
+H5.EnableAddPathAttrs=true
+H5.EnableDropLongString=true
+H5.EnableFillValueCheck=true
+
+H5.EscapeUTF8Attr = true
+H5.EnableCheckNameClashing=false
+H5.NoZeroSizeFullnameAttr=false
+H5.RmConventionAttrPath=true
+H5.KeepVarLeadingUnderscore=false
+H5.CheckIgnoreObj=false
+
+H5.EnablePassFileID=false
+H5.MetaDataMemCacheEntries=1000
+
+H5.EnableDiskMetaDataCache=false
+H5.EnableDiskDataCache=false
+H5.DiskCacheComp=false
+
+H5.DisableStructMetaAttr=true
+H5.DisableECSMetaAttr=false
+H5.EnableEOSGeoCacheFile=false
+----
+
+== Limitations
+
+CF Option: 
+
+* For DAP2, the mappings of 64-bit integer, time, enum, bitfield, opaque, compound, array, and reference types are not supported.  For DAP4, the mapping of HDF5 64-bit integer datatype is supported. But the other datatypes are still unsupported.
```

Comment:
DAP2 doesn't support Int8 (not mentions), DAP4 supports Int8 (not mentioned)

Reply:
Actually Int8 in the DAP2 implementation is mapped to Int16 as shown in the datatype mapping table.  As you said, DAP4 supports Int8. Since this is the limitation section, we don't have to mention this.